src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -222,6 +225,17 @@ function InvoiceDetails({
                 accountingCode={invoice.accountingCode}
               />
             </Accordion>
+            {showVoucherInformation && batchVoucherExport && (
```

Comment:
I think @aliaksei-chumakou  mostly related to this line, if there is no export config - no way to export

Reply:
In my understanding, no export config -> no batch voucher exports.

Reply:
but you can't tell that looking at your code

Reply:
are you sure in it that without export BE can't return batchVoucherExport info?

Reply:
batchVoucherExport is based on exportConfig